core/src/com/unciv/ui/worldscreen/mainmenu/OptionsPopup.kt

```diff
@@ -220,30 +227,38 @@ class OptionsPopup(val previousScreen:CameraStageBaseScreen) : Popup(previousScr
                 generateTranslationsButton.setText("Translation files are generated successfully.".tr())
                 generateTranslationsButton.disable()
             }
-            innerTable2.add(generateTranslationsButton).colspan(2).row()
+            optionsTable.add(generateTranslationsButton).colspan(2).row()
         }
     }
 
-    private fun addModPopup() {
-        val generateTranslationsButton = "Locate mod errors".toTextButton()
-        generateTranslationsButton.onClick {
-            var text = ""
+    private fun addModCheckerPopup() {
+        //if (RulesetCache.isEmpty()) return
+        val modCheckerButton = "Locate mod errors".toTextButton()
+        modCheckerButton.onClick {
+            val lines = ArrayList<String>()
             for (mod in RulesetCache.values) {
                 val modLinks = mod.checkModLinks()
-                if (modLinks != "")
-                    text += "\n\n" + mod.name + "\n\n" + modLinks
+                if (modLinks != "") {
+                    lines += ""
+                    lines += mod.name
+                    lines += ""
+                    lines += modLinks
+                    lines += ""
+                }
             }
+            if (lines.isEmpty()) lines += "{No problems found}."
```

Comment:
Doesn't seem to be in template.properties

Reply:
True. I thought templates are generated? Oh - that's the issue I wanted to mention on the 'Civilopedia sugar' PR - how my invented markup and the translation templates would need some consideration... So do we have documentation on the translation creation and template workflow?